src/css/research.css

```diff
@@ -0,0 +1,856 @@
+@font-face {
+  font-family: 'Syne Neo';
+  src: url('https://storage.googleapis.com/neo4j-fonts/Syne%20Neo%20Medium%20Font.otf') format('opentype');
+}
+
+@media screen and (min-width: 1024px) {
+  body.research main > .content {
+    margin-top: 0;
+  }
+}
+
+@media screen and (min-width: 769px) {
+  body.research .doc .content {
+    margin-top: 0;
+  }
+}
+
+body.research {
+  background: rgba(var(--colors-neutral-15));
+}
+
+/* nav */
+
+body.research .nav-container {
+  display: none;
+}
+
+/* end of nav */
+
+/* collapsible items */
+
+.faq-container {
+  width: 100%;
+  margin: auto;
+}
+
+.faq-item {
+  padding: 10px 0;
+  color: black;
+}
+
+.faq-question {
+  display: flex;
+  color: black;
+}
+
+.faq-answer {
+  display: block; /* Make answers visible by default */
+  padding-bottom: 15px;
+  border-bottom: 1px solid #ccc;
+}
+
+.faq-item .arrow {
```

Comment:
@recrwplay I couldn't get the collapsible drawer to work or use the chevron-up.svg as an icon instead. :/

Reply:
I think `[%collapsible]` not working is a quirk of how the ui-bundle preview is built. If I build this page as part of the docs-home repo I can get the standard collapsible behavior with eg this format to show a list of collapsible entries:

```
.What is a user research panel?
[%collapsible%open]
====
A user research panel is a group of users who have signed up to participate in research studies. 
These studies help us understand how people use our products so we can improve them.
====
```

Reply:
Do you know how we can use an image for the collapsible icon? The instructions I read suggested using the icon in the asciidoc, not in the CSS, but then it would be a special character and not an image.

Reply:
I think if we use it we just have to take the default for now

Reply:
Anu also mentioned we don't really need to make the items collapsible, so we can skip that